### ISSUE TITLE
No automatic password generation

### ISSUE THREAD

**Issue by harpazo64** [FiloSottile/age#117]
## Environment

* OS: MacOS
* age version:

## What were you trying to do
installed with brew
## What happened
i tried the following command
age -p data.txt > data.age

I left the password blank as shown in the readme file, however, no automatic password is generated
```
<insert terminal transcript here>
Enter passphrase:
Error: empty scrypt password
[ Did age not do what you expected? Could an error be more useful? Tell us: https://filippo.io/age/report ]
```

**Comment by Fastidious** [comment]
See #78, I think it relates.

**Comment by sandstrom** [comment]
@harpazo64 can this be closed? (see https://github.com/FiloSottile/age/issues/78)

**Comment by FiloSottile** [comment]
This is fixed in v1.0.0-beta5
